sdk/eventhub/azure-eventhub/azure/eventhub/_producer_client.py

```diff
@@ -105,21 +176,113 @@ def __init__(
         self._max_message_size_on_link = 0
         self._partition_ids = None  # Optional[List[str]]
         self._lock = threading.Lock()
+        self._buffered_mode = buffered_mode
+        self._on_success = on_success
+        self._on_error = on_error
+        self._buffered_producer_dispatcher = None
+        self._max_wait_time = max_wait_time
+        self._max_buffer_length = max_buffer_length
+        self._executor = kwargs.get("executor")
+        self._max_worker = kwargs.get("max_worker")
```

Comment:
We're not documenting these two for now?

Reply:
yeah, I decided to pull it out of our public api surface.

I had left comment in the code as well as opened separate issues for tracking.
we can expose them in the futuer.

Reply:
we? 👀